### ISSUE TITLE
Improve xserver abrt reports

### ISSUE THREAD

**Issue by ofourdan** [abrt/abrt#1416]
Currently, for Xserver crashes, `abrt` is reporting:

- A backtrace, apparently taken from the Xserver's own backtrace feature
- cpuinfo
- dmesg
- dso_list
- etc_X11_xorg_conf_d.tar.gz 
- usr_share_xorg_conf_d.tar.gz

But that's missing an important bit, the actual logs.

Unfortunately, depending on the type of Xserver (`Xorg` or `Xwayland` which are the two main Xservers we're interested in), the logs are not found at the same location...

For Xorg, the `Xorg.<x>.log` (where <x> is the display number) are found either in `/var/log/` or in `~/.local/share/xorg/`

For Xwayland, three is no log file, everything is sent to `stderr` which ends up in `journalctl` under the `gnome-shell` domain (when gnome-shell/mutter is the Wayland compositor, which is the case in Fedora).

Also, previously, the Xserver would catch all crashes and generate the backtrace itself. However, recent versions of the Xserver (1.19) will not prevent a coredump anymore (https://gitlab.freedesktop.org/xorg/xserver/commit/9c7288793) so ideally `abrt` should use that with gdb to get a much better backtrace than the Xserver itself can provide.

So, long story short, I think for the Xserver (either Xorg or Xwayland), we'd need:

- the Xorg logs (even if irrelevant for Xwayland)
- journalctl /usr/bin/gnome-shell aroudnt he time of the crash (like a 5 mins before and after window)
- A gdb backtrace of the crash instead oif the Xserver own backtrace which is incomplete and often misleading.
